### PR TITLE
Manage leagues

### PR DESCRIPTION
**Pull request by dangph-1996**
==> Tittle: Manage leagues

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/16355

### CODE REVIEW

**Comment by dangph-1996** [comment]
ready

**Comment by dangph-1996** [comment]
ready

File: app/views/admin/leagues/_league.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<tr>
+  <td><span><%= get_index_page league_counter %></span></td>
+  <td><%= league.name %></td>
+  <td><%= league.country %></td> 
+  <td><%= league.created_at.strftime(t("format_time")) %></td>
```
Comment (phucmh-1356) [inline app/views/admin/leagues/_league.html.erb]: Dùng I18n để format time

File: app/views/newspapers/new.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+# Newspapers
```
Comment (phucmh-1356) [inline app/views/newspapers/new.html.erb]: k dùng file này thì xóa đi

**Comment by dangph-1996** [comment]
ready

**Pull request merged**